src/app/actions/init.actions.ts

```diff
@@ -1,3 +1,4 @@
 import { createAction } from '../utils/action.utils';
 
-export const initApplication = createAction<void>('INIT_LIBERO');
+export const initApplication = createAction<string>('INIT_LIBERO');
+export const loadArticle = createAction<string>('LOAD_ARTICLE');
```

Comment:
we already have `loadManuscriptAction`. It's an async actions which combines an object of 3 actions **request**, **success** and **error**